File: BankingTool.Model/Dto/BaseDto/DropDownDto.cs
namespace BankingTool.Model
{
    public class DropDownDto
    {
        public int? Key { get; set; }
        public string Value { get; set; }
        public bool IsSelected { get; set; } = false;
    }

    public class DropDownListDto
    {
        public string Name { get; set; }
        public List<DropDownDto> DropDown { get; set; }
    }
}

File: bankingtool.client/src/app/dto/drop-down-dto.ts
export interface DropDownDto {
  key: number;
  value: string;
  isSelected: boolean;
}

export interface DropDownListDto {
  name: string;
  dropDown: DropDownDto[];
}

export interface YesOrNoDto {
  key: boolean;
  value: string;
}
